plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/util/CmsPageUtil.java

```diff
@@ -14,33 +14,30 @@
 package org.entando.entando.plugins.jacms.aps.util;
 
 import com.agiletec.aps.system.SystemConstants;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-
-import org.entando.entando.aps.system.services.widgettype.WidgetTypeParameter;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import com.agiletec.aps.system.services.group.Group;
 import com.agiletec.aps.system.services.page.IPage;
 import com.agiletec.aps.system.services.page.IPageManager;
 import com.agiletec.aps.system.services.page.PageMetadata;
 import com.agiletec.aps.system.services.page.Widget;
-import com.agiletec.aps.system.services.pagemodel.PageModel;
 import com.agiletec.aps.util.ApsProperties;
 import com.agiletec.plugins.jacms.aps.system.JacmsSystemConstants;
 import com.agiletec.plugins.jacms.aps.system.services.content.IContentManager;
 import com.agiletec.plugins.jacms.aps.system.services.content.model.Content;
+import java.util.ArrayList;
+import java.util.Collection;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Properties;
+import org.entando.entando.aps.util.PageUtils;
 import org.entando.entando.plugins.jacms.aps.system.services.content.widget.RowContentListHelper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.context.ApplicationContext;
 
 /**
  * @author E.Santoboni
  */
-public class CmsPageUtil {
+public class CmsPageUtil extends PageUtils {
```

Comment:
Check the build failure

Reply:
Build failure is because there is still a pending PR on entando-core for review.